components/motor/gpio/motor_encoder.go

```diff
@@ -383,6 +383,7 @@ func (m *EncodedMotor) GoFor(ctx context.Context, rpm, revolutions float64, extr
 }
 
 func (m *EncodedMotor) goForInternal(ctx context.Context, rpm, revolutions float64) error {
+	m.RPMMonitorStart()
```

Comment:
Can we check if this atually needs to be started in the constructor? Change the encoders by hand and see if it starts up any motion now or in the case where it is stated in the contructor.

Alos I'd prefer if itwas just the unexported fucntion`rpmmonitor` that stayed in this file.

Reply:
yeah functionality is the same if I only call it here and not in the constructor. will remove from the constructor and unexport

Reply:
actually I think it's easier to follow with RPMMonitorStart, mostly just because of the last 3 lines. it's wrapped into managed go nicely instead of having this other layer inside of rpmMonitor
```
utils.ManagedGo(func() {
		m.rpmMonitor()
	}, m.activeBackgroundWorkers.Done)

Reply:
would it be fine to just un-export `RPMMonitorStart`?

Reply:
or I guess I could also just move that logic to `goforInternal` if you'd prefer that

Reply:
just unexport rpmmonitorstart